sdk/python/kfp/components/pipeline_task.py

```diff
@@ -469,6 +476,39 @@ def my_pipeline():
             self._task_spec.dependent_tasks.append(task.name)
         return self
 
+    def set_exit_task(self, exit_task: 'PipelineTask',
+                      **kwargs) -> 'PipelineTask':
```

Comment:
The function actually returns nothing, I have edited the annotations to reflect that

Reply:
I think we probably want it to return according to the annotation. Typically, public methods on `PipelineTask` return `self` so that we can chain methods (e.g., `t.set_display_name(...).set_exit_task(...)`).